scss/media.scss

```diff
@@ -0,0 +1,428 @@
+@import "vars";
+
+@media (min-width: 1440px) {
+  .wrapper {
+    max-width: 1440px;
+    margin: auto;
+  }
+
+  .gamburgerContainer {
+    display: none;
+    position: absolute;
+  }
+
+  .logo {
+    margin: 28px 84px 0px 84px;
+    .skoutTop {
+      width: 110px;
+      height: 24px;
+    }
+  }
+
+  .containerHeaderMenu {
+    display: flex !important;
+    position: static ;
+    background-color: transparent;
+    flex-direction: row;
+    flex-basis: 78%;
+    justify-content: space-between;
+    .wrapperHeaderMenu {
+      display: flex;
+      flex-basis: 70%;
+    }
+    .headerMenu {
+      flex-direction: row ;
+      margin-top: 0;
+      display: flex;
+      justify-content: space-between;
+      width: 100%;
+      a {
+        color: white ;
+        position: relative;
+        min-width: 85px;
+        &:hover {
+          color: $hoverColorLink;
+        }
+      }
+      .currentLink {
+        &::after {
+          content: "";
+          position: absolute;
+          margin-top: 20px;
+          left: 25%;
+          width: 40px;
+          height: 2px;
+          background-color: white;
+        }
+      }
+    }
+    .enterButton {
+      position: static ;
+      flex-direction: row ;
+      margin-bottom: 0 ;
+      margin-right: 70px;
+      button {
+        margin-top: 20px ;
+        margin-right: 20px ;
+        background-color: transparent;
+        border: 1.5px solid white;
+        color: white ;
+        &:hover {
+          color: $hoverColorLink;
+          border: 1.5px solid $hoverColorLink;
+        }
+      }
+    }
+  }
+
+  .headerText {
+    margin-top: 0;
+    position: absolute;
```

Comment:
this alement shouldn't have position absolute

Reply:
done